app/lib/goods_nomenclature_mapper.rb

```diff
@@ -128,4 +129,11 @@ def parent_of(goods_nomenclature)
   def heading_map?(primary, secondary)
     primary.is_a?(Heading) && secondary.is_a?(Heading)
   end
+
+  def reload_if_stale(goods_nomenclature)
+    # This is a patch to fix the error: "comparison of Integer with nil failed"
```

Comment:
We need to return unless the indents are nil at the start, here

Reply:
done.